test/SignalRServiceExtension.Tests/JobhostEndToEnd.cs

```diff
@@ -120,15 +120,15 @@ public async Task SignalRAttribute_MissingConnectionStringSettingFacts()
         {
             var task = CreateTestTask(typeof(SignalRFunctions), null);
             var exception = await Assert.ThrowsAsync<FunctionInvocationException>(() => task);
-            Assert.Equal(ErrorMessages.EmptyConnectionStringErrorMessageFormat, exception.InnerException.Message);
```

Comment:
why change it?

Reply:
The error hierarchy has changed. Hard coding the exception hierarchy is not resilient to changes.